firefoxThemes/waifus/aqua/theme.json

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
Where did you get these from, the chrome themes?

Reply:
Using the color schemes from both the chrome & edge editions as a reference, I was able to reproduce them for Firefox. The tricky part was Chrome & Firefox have somewhat of a compatibility when it comes to theme creation. [As you see from this chart](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/theme#Colors). 

I took advantage of the theme components that are used exclusively for Firefox for each waifu. Along the way, I also modified and enhanced some the themes to make them stand out more.

Reply:
Thank you for the information!

>modified and enhanced some the themes to make them stand out more


Okay, when you say enhanced did you use the standardized color palette as defined below?

https://github.com/doki-theme/doki-master-theme/blob/33f4fff534925e0df3e35c6b9a0ce33a94d93419/definitions/danganronpa/ibuki/dark/ibuki.dark.master.definition.json#L12

Each manifest file was generated using the master color palette of each theme.

https://github.com/doki-theme/doki-theme-chrome/blob/d2917a5f40af099e9a892e886a999356273c31fe/buildAssets/templates/manifest.template.json#L12

Just want to make sure I take things into account when setting up the the build process for these themes.

Reply:
A few uses the standardized color palette that is defined here, specifically the _bookmark_text_, _ntp_text_, _frame_, & _ntp_background_. 

The other keys however does not. For instance, the _tab_background_text_, _toolbar_, _tab_text_, and everything else has been changed to match the color schemes of each theme.

As [previously mentioned](https://github.com/doki-theme/doki-theme-chrome/pull/13#discussion_r546191540) most of the keys in the [manifest.template.json](https://github.com/doki-theme/doki-theme-chrome/blob/d2917a5f40af099e9a892e886a999356273c31fe/buildAssets/templates/manifest.template.json#L12) are incompatible with Firefox. Additionally, I used theme keys tailored mostly for Firefox, so keep that in mind.

See the [theme format guide](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/theme) on Mozilla for more info on the theme keys.